flash/vision/classification/data.py

```diff
@@ -524,3 +528,52 @@ def from_filepaths(
             seed=seed,
             **kwargs
         )
+
+
+class ImageClassificationDataViz(ImageClassificationData):
+
+    def show_train_batch(self):
+        self.viz.enabled = True
+        # fetch batch and cache data
+        _ = next(iter(self.train_dataloader()))
+        self.viz.enabled = False
+
+        from typing import List
+
+        import kornia as K
```

Comment:
`ImageClassificationData` users `_KORNIA_AVAILABLE` so this should too

Reply:
Removed !